src/deepsparse/yolact/utils/annotate.py

```diff
@@ -0,0 +1,194 @@
+
+    :param image: original image to annotate (no pre-processing needed)
+    :param prediction: predictions returned by the inference pipeline
+    :param images_per_sec: optional fps value to annotate the left corner
+        of the image (video) with
+    :param score_threshold: minimum score a detection should have to be annotated
+        on the image. Default is 0.35
+    :return: the original image annotated with the given bounding boxes
+        (if predictions are not None)
+    """
+
+    masks = prediction.masks[0]
+    boxes = prediction.boxes[0]
+    classes = prediction.classes[0]
+    scores = prediction.scores[0]
+
+    if any(x[0] is None for x in [masks, boxes, classes, scores]):
+        # no detections found
+        return image
+
+    image_res = copy.copy(image)
+
+    masks, boxes = _resize_to_fit_img(image, masks, boxes)
+
+    for box, mask, class_, score in zip(boxes, masks, classes, scores):
+        if score > score_threshold:
+            colour = _get_color(class_)
+            left, top, _, _ = box
+            image_res = _put_mask(image=image_res, mask=mask, colour=colour)
+            image_res = _put_bounding_box(image=image_res, box=box, colour=colour)
+
+            annotation_text = f"{class_}: {score:.0%}"
+            text_width, text_height = _get_text_size(annotation_text)
+            image_res = _put_annotation_text(
+                image=image_res,
+                annotation_text=annotation_text,
+                left=left,
+                top=top,
+                colour=colour,
+                text_width=text_width,
+                text_height=text_height,
+            )
+
+    if images_per_sec is not None:
+        image_res = _plot_fps(
+            img_res=image_res,
+            images_per_sec=images_per_sec,
+            x=20,
+            y=30,
+            font_scale=0.9,
+            thickness=2,
+        )
+
+    return image_res
+
+
+def _put_mask(
+    image: numpy.ndarray, mask: numpy.ndarray, colour: Tuple[int, int, int]
+) -> np.ndarray:
+    img_with_non_transparent_masks = np.where(
+        mask[..., None], np.array(colour, dtype="uint8"), image
+    )
+    img_with_non_transparent_masks = cv2.addWeighted(
+        image, 0.3, img_with_non_transparent_masks, 0.7, 0
+    )
+    return img_with_non_transparent_masks
+
+
+def _put_annotation_text(
+    image: numpy.ndarray,
+    annotation_text: str,
+    colour: Tuple[int, int, int],
+    text_width: int,
+    text_height: int,
+    left: int,
+    top: int,
+) -> numpy.ndarray:
+
+    image = cv2.rectangle(
+        image,
+        (int(left), int(top)),
+        (int(left) + text_width, int(top) + text_height),
+        colour,
+        thickness=-1,
+    )
+
+    image = cv2.putText(
+        image,
+        annotation_text,
+        (int(left), int(top) + text_height),
+        cv2.FONT_HERSHEY_SIMPLEX,
+        0.9,  # font scale
+        (255, 255, 255),  # white text
+        2,  # thickness
```

Comment:
nit: instead of comments could these be made into variables? like `font_scale=0.9`, then use the variable?